examples/MyPackage/CMakeLists.txt

```diff
@@ -9,6 +12,8 @@ facelift_add_qml_plugin(MyPackagePlugin
     URI facelift.example.mypackage
     SOURCES_GLOB_RECURSE ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
     HEADERS_GLOB_RECURSE ${CMAKE_CURRENT_SOURCE_DIR}/*.h
+    HEADERS_GLOB_RECURSE_NO_MOC 
+        ${CMAKE_CURRENT_SOURCE_DIR}/models/cpp/MyInterfaceCppImplementation.h
```

Comment:
this is not GLOB

Reply:
agree, but all the files have been added with next parameter:
`HEADERS_GLOB_RECURSE ${CMAKE_CURRENT_SOURCE_DIR}/*.h`
but if I exclude MyInterfaceCppImplementation.h with 
HEADERS_NO_MOC, I get an error 

> MyPackage/models/cpp/MyInterfaceCppImplementation.h'
  is present in both HEADERS and HEADERS_NO_MOC sections at position '0'

so either we change this example and add the files explicitly, or let it be as is.